userapi/src/main/java/fk/prof/userapi/verticles/HttpVerticle.java

```diff
@@ -265,9 +269,9 @@ else if(result.cause() instanceof IllegalArgumentException) {
     }
 
     private AggregatedProfileNamingStrategy buildFileName(String appId, String clusterId, String procId,
-                                                          AggregatedProfileModel.WorkType workType, String startTime) {
+                                                          AggregatedProfileModel.WorkType workType, String startTime, String duration) {
```

Comment:
pass the duration as integer. String does not make sense for a duration. This method then becomes weird when you have the duration as integer.